.gitignore

```diff
@@ -1,6 +1,7 @@
 /_build
 /deps
 /doc
+/lib/generated
```

Comment:
Can you say more about why this is here for this library package? This might be useful for applications that use this package, but I'm not sure how it works in this context.